.travis.yml

```diff
@@ -41,7 +42,7 @@ install:
 script:
   - vendor/bin/tester $COVERAGE -s -p ${TESTER_RUNTIME:-php} -c ./tests/php.ini-unix ./tests/KdybyTests/
   - php /tmp/php-parallel-lint/parallel-lint.php -e php,phpt --exclude vendor .
-  - if [ "$PHPSTAN" = "1" ]; then php vendor/phpstan/phpstan-shim/phpstan.phar analyse --ansi --no-progress -l7 -c phpstan.neon src tests/KdybyTests; fi
+  - if [ "$PHPSTAN" = "1" ]; then php vendor/bin/phpstan analyse --ansi --no-progress; fi
```

Comment:
You'll also have to update the version of phpstan which gets installed several lines above, otherwise CI is failing.